---a/net/minecraft/block/BlockCompressedPowered.java
+++b/net/minecraft/block/BlockCompressedPowered.java
@@ -7,24 +7,21 @@
 import net.minecraft.util.EnumFacing;
 import net.minecraft.world.IBlockAccess;
 
-public class BlockCompressedPowered extends BlockCompressed
-{
+public class BlockCompressedPowered extends BlockCompressed {
+
     private static final String __OBFID = "CL_00000287";
 
-    public BlockCompressedPowered(MapColor p_i45416_1_)
-    {
+    public BlockCompressedPowered(MapColor p_i45416_1_) {
         super(p_i45416_1_);
         this.setCreativeTab(CreativeTabs.tabRedstone);
     }
 
-    public boolean canProvidePower()
-    {
+    public boolean canProvidePower() {
         return true;
     }
 
-    public int isProvidingWeakPower(IBlockAccess worldIn, BlockPos pos, IBlockState state, EnumFacing side)
-    {
+    public int isProvidingWeakPower(IBlockAccess worldIn, BlockPos pos, IBlockState state, EnumFacing side) {
         return 15;
     }
-}
+}
